webapp/sass/variables.css

```diff
@@ -21,6 +21,7 @@
   --ps-ui-foreground-text: #d8d8d8; /* sidebar text and datepicker text */
   --ps-ui-background: #0e0e13; /* background furthest back color */
   --ps-dropdown-background: #2c2c30; /* dropdown background */
+  --ps-sidebar-separator: #d8d8d833; /* sidebar separation lines color */
```

Comment:
Please add this line also here:

```
[data-theme='light'],
[data-flamegraph-color-mode='light'] {
```

Don't worry about the color we will fix when we do light mode implementation, but this variable just needs to be both under this section and also under the "light mode" section (again it can have the exact same value for now)

Reply:
done